fractions/microprofile/microprofile-fault-tolerance/src/main/java/org/wildfly/swarm/microprofile/faulttolerance/MicroProfileFaultToleranceFraction.java

```diff
@@ -28,4 +31,12 @@
 
     public MicroProfileFaultToleranceFraction() {
     }
+
+    public boolean isSynchronousCircuitBreakerEnabled() {
+        return synchronousCircuitBreaker.get();
+    }
+
+    @AttributeDocumentation("Enable/disable synchronous circuit breaker functionality. If disabled, `CircuitBreaker#successThreshold()` of value greater than 1 is not supported. Moreover, circuit breaker does not necessarily transition from `CLOSED` to `OPEN` immediately when a fault tolerance operation completes. However, applications are encouraged to disable this feature on high-volume circuits.")
+    @Configurable("swarm.micro-profile-faulttolerance.synchronousCircuitBreaker")
```

Comment:
Oh! We're crazy inconsistent in how MP fractions name their config keys :-(

Reply:
Yes, we do. Pls create a separate issue. Thanks.

Reply:
Sure, filed https://issues.jboss.org/browse/SWARM-1963